tests/test_mgxs_library_no_nuclides/results_true.dat

```diff
@@ -129,18 +129,18 @@
 8      10000             5         2   total  0.004684   0.000451
 10     10000             6         2   total  0.001962   0.000189
     material  delayedgroup  group in nuclide      mean  std. dev.
-1      10000             1         1   total  0.000000   0.000000
-3      10000             2         1   total  0.000000   0.000000
-5      10000             3         1   total  0.000000   0.000000
-7      10000             4         1   total  0.000000   0.000000
-9      10000             5         1   total  0.000000   0.000000
-11     10000             6         1   total  0.000000   0.000000
-0      10000             1         2   total  0.000000   0.000000
-2      10000             2         2   total  0.032739   0.028454
-4      10000             3         2   total  0.120780   0.170809
-6      10000             4         2   total  0.302780   0.109110
-8      10000             5         2   total  0.000000   0.000000
-10     10000             6         2   total  0.000000   0.000000
+1      10000             1         1   total  0.013445   0.001084
+3      10000             2         1   total  0.032064   0.002658
+5      10000             3         1   total  0.122136   0.010296
+7      10000             4         1   total  0.315269   0.027175
+9      10000             5         1   total  0.889233   0.079368
+11     10000             6         1   total  2.989404   0.266253
```

Comment:
This is a pretty big score to have been missed by using analog estimators. Is that really all thats going on here?